gui/launcher.lua

```diff
@@ -441,13 +438,15 @@ function LauncherUI:init(args)
     }
 
     local update_frames = function()
-        local new_frame = {l=5, r=5}
+        local new_frame = {}
         if self.minimal then
             new_frame.l = 0
-            new_frame.r = 0
+            new_frame.r = 58
```

Comment:
Is this just for viewscreen_dwarfmodest? Wondering if we can special-case that easily. It looks a bit out of proportion on other screens to me:

![image](https://user-images.githubusercontent.com/3719547/210121797-662e33b9-5ec3-496c-9d60-e2bb43a88c08.png)
![image](https://user-images.githubusercontent.com/3719547/210121802-8d6824a8-1a73-4736-93b3-c3eedd905ff6.png)

Reply:
Or could we somehow detect things in the anchored layer somehow? (does that require that the tiles are the same size as the text/UI layer?)

Reply:
luckily, since this is on the top row, we can detect anchored layer textures fairly easily, since they only scale down and to the right. I'll see if I can code that up.